src/content/docs/products/splashkit/SplashKit Tutorials/Documentation/3-Tutorial-Reviews.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Tutorial Reviews
+sidebar:
+  label: 3. Tutorial Reviews
+  order: 3
+---
+
+## Filling out the Tutorial Review
+
+After completing a tutorial review card. Please fill out the information under the designated spot on this page for the tutorial, using the following template:
+
+### Review Template
+
+#### Description:
+
+This tutorial covers how to do this...
+
+Website Link:
+
+Repo Link:
+
+#### Confirmation of Passing Quality Checks:
+
+- [ ] The tutorial is free of spelling and grammatical errors.
+- [ ] The tutorial is easy to follow.
+- [ ] The tutorial uses top level statements in C# and C++.
+- [ ] The code passes all the Quality Checks.
+
+#### Result of the code:
+
+Add the results here...
```

Comment:
Possibly provide an example of what a "result of code" looks like. Are we looking for outputs? Errors? Compatibility with different scenarios?